ex4/server/routes/api.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
```

Comment:
Nice!

Reply:
If you rename the file `api.js` I would assume that the route will be `localhost/api/items`, please rename the file or the routing